android/libraries/rib-base/src/main/java/com/badoo/ribs/android/RequestCodeBasedEventStream.kt

```diff
@@ -0,0 +1,14 @@
+package com.badoo.ribs.android
+
+import com.badoo.ribs.android.RequestCodeBasedEventStream.RequestCodeBasedEvent
+import com.badoo.ribs.core.Identifiable
+import io.reactivex.Observable
+
+interface RequestCodeBasedEventStream<T : RequestCodeBasedEvent> {
```

Comment:
How about renaming this and related classes to a bit shorter name?
`RequestCodeBasedEvent` sounds a bit too verbose.

Reply:
Suggestions please

Reply:
For example - `RequestCodeEvent`

Reply:
That's semantically not correct. I'll leave it for now as is.